### PR TITLE
Translation handle for dice_helper

### PR DESCRIPTION
**Pull request by ercete**
Replacing & handling i18n skill names.
Searching if a dice_helper_LANG.json instead of generic one.

### CODE REVIEW

File: content/dice_helper.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "brawl": {
```
Comment (wrycu) [inline content/dice_helper.json:2]: Hmm, I'm not sure about this one. I'm pretty sure it broke at some point (I originally had it the way you're now changing it to)

**Review by wrycu — COMMENTED** [review]
left comments

File: content/dice_helper.json (hunk 1 above)
Comment (ercete) [reply content/dice_helper.json:2]: I made several tests with original one and translated one.
Also by changing Foundry LANG.
It needs at least a journal delete and a reload to take fully effect.

Comment (wrycu) [reply content/dice_helper.json:2]: To be clear, are you saying this is a breaking change for anyone already running the module?

Comment (ercete) [reply content/dice_helper.json:2]: No, I made it not breaking.
If the skillname in journal starts with "SWFFG." then the skill translation is searched and loaded for building the chat message. But with an old journal (or a handmade one for example) the skillname is loaded as it.
A user with an handmade journal in another langage who has manually tranlated his skills should still work.
But a new user with a foundry configured with another lang can now see the chatmessages (EN by default, FR if set).

**Pull request merged**